### ISSUE TITLE
Update fusion install script to build the install package

### ISSUE THREAD

**Issue by tst-lsavoie** [google/earthenterprise#109]
Currently, installing fusion requires running the following commands:

```
cd src
scons -j3 release=1 installdir=/tmp/fusion_os_install install
cd ..
scons -j3 installdir=/tmp/fusion_os_install install
cd src/installer
sudo ./install_fusion.sh
```

These steps should be combined so that we can install fusion by running a single script.

**Comment by arjntc1** [comment]
I would hold off on this.  The install and the building of the install package needs to remain separate.  We cannot assume that the machine on which we are installing is the same on which the code will be built.

**Comment by tst-ccamp** [comment]
One thing I do think we should change soon, though, is to make the default installdir for scons match the default specified in the install scripts. I think I'll open a new ticket for that.

**Comment by tst-lsavoie** [comment]
The point of this ticket is that it's confusing to run scons for install, then go up a directory, then run scons for install again.  If we could combine just that part into a single step, that would go a long way to helping new people.

**Comment by arjntc1** [comment]
I like @tst-ccamp's idea of defaulting to the same "installdir" as the installers.  @tst-lsavoie Combining the three build install steps into a single "build install" command would also be very helpful.  The initial comment didn't state that though -- it (and the title of this issue) refer to combining the building of the install package with the installer (slightly different meaning).  I actually think a single "build install" command (separate from the installer) would be a plus.

**Comment by tst-lsavoie** [comment]
That's true - perhaps we should rewrite this ticket or close it and open another one so that the description is right.

**Comment by tst-lsavoie** [comment]
Closed this and opened #131 with a better title/description.
